tests/10-rdfc-create.js

```diff
@@ -158,6 +222,12 @@ describe('eddsa-rdfc-2022 (create)', function() {
             }
           }
         });
+        /**
+         * Sanity check to ensure that the verifier is capable of
+         * verifying the result.
+         *
+         * This assertion is not directly related to the specification.
+         */
         it('"proof" MUST verify when using a conformant verifier.',
```

Comment:
this seems fair, but I'm pretty confident something in the spec has a must that references a section related to verification.

Reply:
Ill look closer